app/controllers/api/v1/stageaction/result.html.erb

```diff
@@ -51,4 +51,5 @@
       </div>
     </section> 
   </section>
+</main>
```

Comment:
@lgalis The "main" tag for both show and result is closed in the footer. This need needs to be removed.